plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -36,6 +37,7 @@
     public static final String AGGREGATION_PUSHDOWN_ENABLED = "aggregation_pushdown_enabled";
     public static final String TOPN_PUSHDOWN_ENABLED = "topn_pushdown_enabled";
     public static final String DOMAIN_COMPACTION_THRESHOLD = "domain_compaction_threshold";
+    public static final String INSERT_BATCH_SIZE = "insert_batch_size";
```

Comment:
same here -- it shouldn't be in this class, since it's not consumed by metadata layer